master/internal/rm/kubernetesrm/kubernetes_resource_manager_intg_test.go

```diff
@@ -851,6 +853,142 @@ func TestVerifyNamespaceExists(t *testing.T) {
 	require.Error(t, err)
 }
 
+func TestCreateNamespaceHelper(t *testing.T) {
+	js := createMockJobsService(nil, device.CPU, false)
+
+	validNamespace := "valNamespace"
+	existentNamespace := "nonexistentNamespace"
```

Comment:
I think we should rename this var `nonExistentNamespace`

Reply:
done